.github/workflows/brakeman.yml

```diff
@@ -2,11 +2,6 @@ name: brakeman
 
 on:
   push:
-    branches:
```

Comment:
This and the ones like it run brakeman and linting and such on all pushes/branches, not only `main`

Reply:
This should be reverted before merge, no?

Reply:
I think we should keep them -- they then will run the checks on any sub-branch PR which is what I expected.

Reply:
All righty.

Reply:
Just realized - removing `pull_request` means that it'll run on the branch but _not_ on the merge commit. If the main branch has been updated, these runs will miss those commits.

Reply:
OK -- I tried to make it run on PR and consistently across all the jobs. Also updated the plantuml gen a bit. Here is the result -- https://github.com/rubyforgood/human-essentials/pull/3788/commits/aba16d19f904d209412f0690f22426c562fc04e4